src/main/java/core/basesyntax/Palindrome.java

```diff
@@ -21,6 +21,9 @@
      * <p>Результат: true</p>
      */
     public boolean isPalindrome(String text) {
-        return false;
+        String newText = text.replaceAll("\\p{Punct}+", "").replaceAll(" ","")
```

Comment:
we can do it calling replaceAll only once, try to think how to do it

Reply:
done